dist/index.html

```diff
@@ -54,10 +54,10 @@
 
       <button id="startButton">Play</button>
       <div id="difficultyButtons" class="buttonContainer">
-        <button id="normalButton" class="difficultyButton" onclick="setDifficulty('normal')">
+        <button id="normalButton" class="difficultyButton">
```

Comment:
If you have a function named setDifficulty in a JavaScript module and you're trying to call it from an onclick attribute in your HTML, it will not work because functions declared in a module are not in the global scope, which means they're not accessible directly from HTML inline event handlers.

ChatGPT :P

Reply:
Oops i see, but hey, atleast it still worked :P I'll keep it in mind though, thanks for the tip!